save-backend/src/main/kotlin/com/saveourtool/save/backend/service/vulnerability/VulnerabilityService.kt

```diff
@@ -115,39 +113,26 @@ class VulnerabilityService(
 
     /**
      * @param filter
-     * @param authentication
      * @return count of vulnerability
      */
     fun getCountVulnerabilitiesByFilter(
         filter: VulnerabilityFilter,
-        authentication: Authentication?,
-    ) = vulnerabilityMetadataRepository.count { root, cq, cb -> getFilterPredicate(root, cq, cb, filter, authentication) }
+    ) = vulnerabilityMetadataRepository.count { root, cq, cb -> getFilterPredicate(root, cq, cb, filter) }
 
     private fun getFilterPredicate(
         root: Root<VulnerabilityMetadata>,
         cq: CriteriaQuery<*>,
         cb: CriteriaBuilder,
         filter: VulnerabilityFilter,
-        authentication: Authentication?,
     ): Predicate {
         with(filter) {
-            val namePredicate = if (identifierPrefix.isBlank()) {
-                cb.and()
+            val namePredicate = if (identifierOrSummaryPrefix.isNotBlank()) {
```

Comment:
it's not a prefix anymore, also we can search by description too and suggest to rename it to `freeText` (or something like it)

Reply:
I think that it may not be clear to user if search result shows vulnerabilities that we found by `Details`, because this field isn't displayed in the table.

Reply:
agree, but still suggest to rename to `freeText`

Reply:
Change `freeText` for property in `VulnerabilityFilter` and `FreeText...` for input field? Will it be clear to user that search is performed by `Identifier` and `Summary`? Or only `freeText` for property?

Reply:
discussed offline